src/frontend/magnify/src/context/auth/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useQueryClient } from '@tanstack/react-query';
+import React, { useEffect, useState } from 'react';
+import { Maybe } from 'yup/es/types';
+import { User } from '../../types/entities/user';
+import { MagnifyQueryKeys } from '../../utils/constants/react-query';
+
+export interface AuthContextInterface {
+  user: Maybe<User> | null;
+  updateUser: (user: Maybe<User> | null) => void;
```

Comment:
ditto ( and anywhere ` | null` is used

Reply:
Yes I understood the usefulness of Maybe, but this time null has a utility, to allow the user to have 3 states